docs/src/reference/mount.md

```diff
@@ -740,12 +775,16 @@ Starts the mount server without performing the OS-level mount. This is intended
 everest mount-server <remote_mount_uri> [flags]
 ```
 
+!!! note "Windows Support"
+    Windows users must utilze the `--root` flag
+
```

Comment:
no need in that. 
`mount-server` is an advanced command anyway. 
If someone want's to use it there are many gotchas we don't mention. 
Just keep it simple with the explanation you already added below for the flag `--root`